test/source/mock/backend/backend-endpoints.ts

```diff
@@ -58,16 +46,36 @@ export const mockBackendEndpoints: HandlersDefinition = {
   },
 };
 
-const throwIfNotPostWithAuth = (body: unknown, req: IncomingMessage) => {
-  const parsed = body as Dict<any>;
+interface OpenId {
+  name: string;
+  email?: string;
+  email_verified?: boolean;
+}
+
+const throwIfNotPost = (req: IncomingMessage) => {
   if (!isPost(req)) {
     throw new HttpClientErr('Backend mock calls must use POST method');
   }
-  if (!parsed.account) {
-    throw new HttpAuthErr('Backend mock call missing value: account');
+};
+
+const throwIfIdTokenIsInvalid = (req: IncomingMessage) => {
+  const idToken = req.headers.authorization?.replace(/^Bearer /, '');
+  if (!idToken) {
+    throw new HttpClientErr('backend mock: Missing id_token');
   }
-  if (!parsed.uuid) {
-    throw new HttpAuthErr('Backend mock call missing value: uuid');
+  if (!oauth.isIdTokenValid(idToken)) {
+    throw new HttpAuthErr(`Could not verify mock idToken: ${idToken}`);
   }
-  return parsed;
+  return parseIdToken(idToken);
 };
+
+const parseIdToken = (idToken: string): OpenId => {
+  const claims = JSON.parse(Buf.fromBase64UrlStr(idToken.split(/\./g)[1]).toUtfStr()) as OpenId;
+  if (claims.email) {
+    claims.email = claims.email.toLowerCase();
+    if (!claims.email_verified) {
```

Comment:
Do we ever have a situation in tests where `email_verified === false`?
I haven't found anything.

Reply:
Oh it's just a copy/paste from `GoogleAuth.parseIdToken` ...

Reply:
It's just copy/paste. i also think there will not be a situation where email_verified is false